src/js/graph.js

```diff
@@ -0,0 +1,60 @@
+( function( $, window, document ) {
+
+	/**
+	 * Check if the graph is in the viewport.
+	 */
+	var in_viewport = function( graph ) {
+		if ( graph instanceof jQuery ) {
+			graph = graph[ 0 ];
+		}
+
+		var rect = graph.getBoundingClientRect();
+
+		return rect.bottom > 0 &&
+			rect.right > 0 &&
+			rect.left < ( window.innerWidth || document.documentElement.clientWidth ) &&
+			rect.top < ( window.innerHeight || document.documentElement.clientHeight );
+	};
+
+	/**
+	 * Animate the graph when it is scrolled into the viewport.
+	 */
+	var animate_graph = function() {
+		var $graph = $( ".bargraph" ),
+			total = parseInt( $graph.data( "total" ).replace( /,/g, "" ), 10 ),
+			prefix = $graph.data( "prefix" );
+
+		$( document ).on( "scroll", function() {
+			if ( !in_viewport( $graph ) || $graph.hasClass( "animated" ) ) {
+				return;
+			}
+
+			$graph.addClass( "animated" ).find( "li" ).each( function( i ) {
+				var $bar = $( this ),
+					value = parseInt( $bar.find( ".value" ).html().split( prefix ).pop().replace( /,/g, "" ), 10 ),
```

Comment:
It could be worth considering a data attribute on each list element that contains the number value without commas, etc..

`<li data-dollar-amt="100000000">....</li>`

That would make this part easier to parse. (I think)